CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
 # Upcoming release
 
 ### Added
+- **Encodable**
+- Added  `encode(_:to:)` method to save Encodable object to file at provided URL. [#598](https://github.com/SwifterSwift/SwifterSwift/pull/598) by [AlexeiGitH](https://github.com/AlexeiGitH)
```

Comment:
I know I'm being pedantic but... 😛 
```suggestion
- Added `encode(_:to:)` method to save Encodable object to file at provided URL. [#598](https://github.com/SwifterSwift/SwifterSwift/pull/598) by [AlexeiGitH](https://github.com/AlexeiGitH)
```

Reply:
Fixed. It's so easy to miss because of that `